### PR TITLE
Flutter Global Summit

### PR DESCRIPTION
**Pull request by ueman**

### CODE REVIEW

File: _conferences/flutter-global-summit-february-2024.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+website: https://events.geekle.us/flutter24/
+location: Online
+online: Online
+
+date_start: 2024-02-06
+date_end:   2023-02-07
+
+cfp:
+  start: 2023-12-01
+  end:   2024-02-01
```
Comment (ueman) [inline _conferences/flutter-global-summit-february-2024.md:12]: There's no end date given 🤷‍♂️

File: _conferences/flutter-global-summit-february-2024.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Flutter Global Summit"
+website: https://events.geekle.us/flutter24/
+location: Online
+online: Online
```
Comment (ueman) [inline _conferences/flutter-global-summit-february-2024.md]:
```suggestion
online: true
```

File: _conferences/flutter-global-summit-february-2024.md (hunk 1 above)
Comment (marandaneto) [reply _conferences/flutter-global-summit-february-2024.md:12]: We can fix it once we know =)

**Comment by ueman** [comment]
@marandaneto For some reason this conference is listed under past conferences: https://www.flutterconferences.com/past.html but not in the upcoming ones: https://www.flutterconferences.com/

Do you know why?

**Comment by marandaneto** [comment]
> @marandaneto For some reason this conference is listed under past conferences: [flutterconferences.com/past.html](https://www.flutterconferences.com/past.html) but not in the upcoming ones: [flutterconferences.com](https://www.flutterconferences.com/)
> 
> Do you know why?

probably https://github.com/marandaneto/flutter-conferences/commit/a84a2c79568b05e6818cec56bc12b5e97432e076

**Comment by ueman** [comment]
Oh, good catch

**Pull request merged**